include/database_functions.php

```diff
@@ -1,92 +1,145 @@
 <?php
+// *********************************************************************************************************************************
+// Admins table
+// *********************************************************************************************************************************
+
+function getAllAdmins(){
+        return dbQuery("
+        SELECT *
+        FROM admins 
+        WHERE DateArchived is NULL 
+        ORDER BY AdminId     
+    ")->fetchAll(); 
+}
+
+// *********************************************************************************************************************************
+
+function getOneAdmin($adminId){
+    return dbQuery("
+        SELECT *
+        FROM admins
+        WHERE AdminId = $adminId
+        AND DateArchived is NULL     
+    ")->fetch(); 
+}
 
+// *********************************************************************************************************************************
 //Students Table
-// --------------------------------------------------------------------------
+// *********************************************************************************************************************************
 
 function getAllStudents(){
     return dbQuery("
             SELECT *
-            FROM students
+            FROM students 
             WHERE DateArchived is NULL 
             ORDER BY StudentId     
     ")->fetchAll();   
 }
 
+// *********************************************************************************************************************************
+//I will ask this in next jog but do I need to clean studentid here to prevent sql injection as well?
```

Comment:
Yep! In general, any data you use to run a query should be cleaned in some way to prevent sql injection.